.gitignore

```diff
@@ -99,3 +99,5 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+MEDIA/
```

Comment:
I don't think this is working because I can see you've uploaded images from your media file